apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -313,7 +304,51 @@ def _find_modified_include_path(self, project_dir, file_path, include_path):
         # It's probably a standard C/C++ header
         return include_path
 
+    def _cmsis_required(self, project_path: Union[str, pathlib.Path]) -> bool:
```

Comment:
Could we standardize this to only accept a `pathlib.Path`? Since `_cmsis_required` is an internal function, doing so seems cleaner.
```suggestion
    def _cmsis_required(self, project_path: pathlib.Path) -> bool:
```